docs/src/content/docs/hi/404.md

```diff
@@ -0,0 +1,13 @@
+---
+title: पेज नहीं मिला
+template: splash
+editUrl: false
+hero:
+  title: '404'
+  tagline: <strong>Houston, हमें एक समस्या है।</strong> हमें वह पेज नहीं मिला.<br> URL जांचें या खोज बार का उपयोग करने का प्रयास करें।
+  actions:
+    - text: अवतरण पेज पर वापस आये
+      icon: right-arrow
+      link: /
```

Comment:
Looks great — thank you @at-the-vr!

One last thing, the link should point to the Hindi homepage here:

```suggestion
      link: /hi/
```